applications/nrf5340_audio/src/modules/lc3_file.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "lc3_file.h"
+#include "sd_card.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(sd_card_lc3_file, CONFIG_MODULE_SD_CARD_LC3_FILE_LOG_LEVEL);
+
+int lc3_file_frame_get(struct lc3_file_ctx *file, uint8_t *buffer, size_t buffer_size)
+{
+	int err;
+
+	/* Read frame header */
+	uint16_t frame_header;
+	size_t frame_header_size = sizeof(frame_header);
+
+	err = sd_card_read((char *)&frame_header, &frame_header_size, &file->file_object);
+	if (err) {
+		LOG_ERR("Failed to read frame header: %d", err);
+		return err;
+	}
+
+	if ((frame_header_size == 0) || (frame_header == 0)) {
+		LOG_ERR("No more frames to read");
+		return -ENODATA;
+	}
+
+	LOG_DBG("Size of frame is %d", frame_header);
+
+	/* Read frame data */
+	size_t frame_size = frame_header;
+
+	err = sd_card_read((char *)buffer, &frame_size, &file->file_object);
+	if (err) {
+		LOG_ERR("Failed to read frame data: %d", err);
+		return err;
+	}
+
+	if (frame_size != frame_header) {
+		LOG_ERR("Frame size mismatch: %d != %d", frame_size, frame_header);
+		return -EIO;
+	}
+
+	return 0;
+}
+
+int lc3_file_open(struct lc3_file_ctx *file, const char *file_name)
+{
+	int err;
+	size_t size = sizeof(file->lc3_header);
+
+	err = sd_card_open(file_name, &file->file_object);
+	if (err) {
+		LOG_ERR("Failed to open file: %d", err);
+		return err;
+	}
+
+	/* Read LC3 header and store in struct */
+	err = sd_card_read((char *)&file->lc3_header, &size, &file->file_object);
+	if (err) {
+		LOG_ERR("Failed to read the LC3 header: %d", err);
+		return err;
+	}
+
+	if (file->lc3_header.file_id != 0xcc1c) {
```

Comment:
Create a define for this ID. Is this ID unique to the LC3 tool we use, all will it be the same for all LC3 PC tools?